src/client/lib/utils.js

```diff
@@ -44,7 +44,7 @@ export var clone = (obj) => deserialize(serialize(obj));
  * @param  {String} key The string descriptive of the value path
  * @return {Value || undefined}
  */
-export var lookup = (data, key) => {
+export let lookup = (data, key) => {
```

Comment:
'export' is only available in ES6 (use esnext option).<br>'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).